addons/portal/static/src/js/crash_manager.js

```diff
@@ -0,0 +1,50 @@
+odoo.define('portal.CrashManager', function (require) {
+"use strict";
+
+var core = require('web.core');
+var CrashManager = require('web.CrashManager').CrashManager;
+var Notification = require('web.Notification');
+var AbstractService = require('web.AbstractService');
+
+var _t = core._t;
+
+/**
+ * An extension of Notification Widget to render the notifications on the website.
+ * @override
+ */
+var CrashManagerNotification = Notification.extend({
+    xmlDependencies: (Notification.prototype.xmlDependencies || []).concat(
+        ['/web/static/src/xml/notification.xml']
+    ),
+    template: 'Notification',
+});
+
+CrashManager.include({
+    show_toast: function (error) {
```

Comment:
Javascript, not python ;) => Camelcase
Also, should not this be part of the standard crash manager instead of doing an include ? If not, maybe use an extend instead of an include.

Reply:
I have done it this way so that these changes are only applying to website. I dont really see how to extend ```CrashManager``` and make it work...